src/components/Layout.tsx

```diff
@@ -30,7 +37,73 @@ export function Layout({ children }: LayoutProps) {
           {children}
         </Grid>
         <Grid item xs={12}>
-          <Footer />
+          <Footer
```

Comment:
Could you restore the Footer.tsx file and write an own Footer component that return the mui Footer component configured like that?